app/services/send_sign_up_email_confirmation.rb

```diff
@@ -12,6 +12,8 @@ def call(request_id: nil, instructions: nil, password_reset_requested: false)
 
     if password_reset_requested && !user.confirmed?
       send_pw_reset_request_unconfirmed_user_email(request_id, instructions)
+    elsif user.suspended?
+      send_suspended_user_email
```

Comment:
Is this alright for logic placement?  Or should it be the first check?

Reply:
I can't think of a way to have an unconfirmed user be suspended.... but it wouldn't hurt to move this first